packages/client/src/object/fetchPageOrThrow.ts

```diff
@@ -69,13 +70,17 @@ export async function fetchPageOrThrow<
   const body: LoadObjectSetRequestV2 = {
     objectSet,
     // We have to do the following case because LoadObjectSetRequestV2 isnt readonly
-    select: (args?.select ?? []) as unknown as string[], // FIXME?
+    select: ((args?.select as string[] | undefined) ?? []), // FIXME?
   };
 
   if (args?.nextPageToken) {
     body.pageToken = args.nextPageToken;
   }
 
+  if (args?.pageSize != null) {
+    body.pageSize = args.pageSize;
+  }
```

Comment:
Was there a reason this wasn't exposed before?

Reply:
Mistake on my part. Not intentional